helpers/import.sh

```diff
@@ -155,30 +155,30 @@ terraform import "module.$MODULE_LOCAL_NAME.google_project_service.main[15]" "$P
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[0]" "$ORG_ID roles/appengine.appViewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[1]" "$ORG_ID roles/bigquery.metadataViewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[2]" "$ORG_ID roles/browser serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[3]" "$ORG_ID roles/cloudasset.viewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[4]" "$ORG_ID roles/cloudsql.viewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[5]" "$ORG_ID roles/compute.networkViewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[6]" "$ORG_ID roles/iam.securityReviewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[7]" "$ORG_ID roles/orgpolicy.policyViewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[8]" "$ORG_ID roles/servicemanagement.quotaViewer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_read[9]" "$ORG_ID roles/serviceusage.serviceUsageConsumer serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
-terraform import "module.$MODULE_LOCAL_NAME.module.server.google_organization_iam_member.org_write[0]" "$ORG_ID roles/compute.securityAdmin serviceAccount:forseti-server-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
+terraform import "module.$MODULE_LOCAL_NAME.module.client_iam.google_project_iam_member.client_roles[0]" "$PROJECT_ID roles/storage.objectViewer serviceAccount:forseti-client-gcp-$RESOURCE_NAME_SUFFIX@$PROJECT_ID.iam.gserviceaccount.com"
```

Comment:
Is there another client role (`roles/cloudtrace.agent`) that should be imported?  https://github.com/forseti-security/terraform-google-forseti/blob/44f894cc73a5a8a91d8cb54aab7afd39483ba494/modules/client_iam/main.tf

Reply:
Not necessarily.  This isn't in the master branch.

If the role binding isn't imported and then `terraform apply` is run, It should reconcile.

Reply:
I would suggest we drop role binding imports entirely. Importing them isn't necessary because recreating a role binding is a no-op.

Reply:
nm.  I think you are correct.  It should be imported.

Reply:
@morgante okeedokee.  That makes it simpler.